ansible_events_ui/db/models.py

```diff
@@ -170,6 +171,21 @@
     ),
     sqlalchemy.Column("git_hash", sqlalchemy.String),
     sqlalchemy.Column("url", sqlalchemy.String),
+    sqlalchemy.Column("name", sqlalchemy.String),
+    sqlalchemy.Column("description", sqlalchemy.String),
+    sqlalchemy.Column(
+        "created_at",
+        sqlalchemy.DateTime(timezone=True),
+        nullable=False,
+        server_default=func.now(),
+    ),
+    sqlalchemy.Column(
+        "modified_at",
+        sqlalchemy.DateTime(timezone=True),
+        nullable=False,
+        server_default=func.now(),
```

Comment:
To the previous comment regarding consistency, the #68 uses nullable `last_modified` field with `NULL` as a default, while here `now()` is used as a default.

Reply:
Would we want it to default to `NULL` from a UI perspective? 

on creation we end up at the project detail page and we would have a section for `last_modified` that will remain blank until a user edits the project, or if we default it would match creation time until it is updated. I can change it to `NULL` default just didn't know which we preferred.